### ISSUE TITLE
Displays bugs for readonly/deprecated projects

### ISSUE THREAD

**Issue by douglasg14b** [jdm/bugsahoy#187]
It's unhelpful for this tool to show bugs for projects that can't accept contributions or are deprecated. 

https://github.com/mozilla/memchaser is an example

**Comment by jdm** [comment]
Agreed! Please feel free to make pull requests like https://github.com/jdm/bugsahoy/commit/e3a674f16725312831145e22dee8c7c8327aa054 to remove any other projects that are no longer appropriate to display.
